ldk/javascript/examples/self-test-loop/src/tests/config/index.ts

```diff
@@ -0,0 +1,43 @@
+import { config } from '@oliveai/ldk';
+
+export const testLoopConfig = (): Promise<boolean> =>
+  new Promise((resolve, reject) => {
+    config
+      .getLoopConfig()
+      .then((loopConfig) => {
+        console.log(loopConfig);
+
+        if (loopConfig instanceof Object) {
+          console.log('Success: loopConfig is an object');
+
+          const EXPECTED_VARIABLE_NAMES = [
+            'testBoolean',
+            'testNumber',
+            'testNumberArray',
+            'testNumberEnum',
+            'testObject',
+            'testString',
+            'testStringArray',
+            'testStringEnum',
+          ];
+
+          const actualVariableNames = Object.keys(loopConfig).sort();
+
+          if (JSON.stringify(EXPECTED_VARIABLE_NAMES) === JSON.stringify(actualVariableNames)) {
+            console.log('Success: loopConfig keys match expected schema');
+            resolve(true);
```

Comment:
I never know if it's **needed**, but I think a `return` after this line would be "good" to include

Reply:
Just for you, buddy